spec/test_app/app/controllers/application_controller.rb

```diff
@@ -1,3 +1,7 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  def update_enrollment_last_activity_at
+    #
```

Comment:
What's this about? Seems strange to have a method here that doesn't actually do anything

Reply:
This is to make the tests pass in test_app